src/OSPSuite.Core/Domain/Services/IDataRepositoryTask.cs

```diff
@@ -14,19 +15,24 @@ public interface IDataRepositoryTask
       ///    Returns one datable for each base grid and associated columns representing the content of the repository (one column
       ///    for each DataColumn)
       ///    The column headers will be set to the column name();
-      ///    If the flag <paramref name="formatOutput" /> is set to true, the value will be formatted using the number of decimal
+      ///    If the flag <paramref name="formatOutput" /> is set to <c>true</c> (default is <c>false</c>), the value will be formatted using the number of decimal
       ///    places defined in the application
+      ///    If the flag <paramref name="useDisplayUnit" /> is set to <c>true</c> (default is <c>true</c>), the value will be exported in the display unit of the 
```

Comment:
I have added this flag (default is true to keep the same behavior as before), that specifies whether the display unit should be used or not